### PR TITLE
chore(i18n): Adds limited support for new languages on Transifex

### LINKED ISSUE #7540: Transifex breaks sprintf format/dollar signs in translations
Change lang file name of portuguese files to "pt_br" (currently this as pt_BR). for this reason the translation is not working.

### PR DESCRIPTION
**Pull request by mrclay**
Updates release script to normalize language filenames and remove unrecognized language files. Adds missing InvalidLocaleException.

Fixes #7540

(see also #8687, which this replaces)

### CODE REVIEW

**Comment by mrclay** [comment]
Here I just manually add the few new translations we have in Transifex, but basically translators would need to come to us to approve new langs. I think this is OK short term.

**Comment by iionly** [comment]
There are also translations for eu_ES ( Basque (Spain) ) available at Transifex. So, you might want to add "eu_es" as allowed translation, too.

File: .scripts/transifex/scrape.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/**
+ * This is just a manual process for updating the languages available from Transifex
+ */
```
Comment (juho-jaakkola) [inline .scripts/transifex/scrape.js]: Can you add a longer docblock description that tells what exactly the script does and why? Inline comments would also help.

**Comment by mrclay** [comment]
I added "eu_es" and got rid of the scrape.js file. Not really necessary to keep it.

**Comment by juho-jaakkola** [comment]
Haven't tested but looks good.

**Comment by juho-jaakkola** [comment]
For some reason this drops ```it.php``` but our current 2.0 HEAD doesn't. I don't understand why.

**Comment by juho-jaakkola** [comment]
Oops, my bad. This is for 1.12, not 2.0. :)

**Comment by juho-jaakkola** [comment]
Yep, Italian is only 98% done on Transifex 1.x project. That's what caused it to drop.

**Comment by juho-jaakkola** [comment]
Issue to discuss the translation completeness: #8959

**Pull request merged**